src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -59,8 +59,6 @@ public static void SetPermissionsForQueue(MessageQueue queue, string account)
         {
 
             queue.SetPermissions(LocalAdministratorsGroupName, MessageQueueAccessRights.FullControl, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalEveryoneGroupName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
```

Comment:
@andreasohlund ok, I'll revert that commit and add a comment based on Matt his detailed feedback.